### ISSUE TITLE
Incompatible with Stripe API version November 16, 2022 and newer

### ISSUE THREAD

**Issue by marckohlbrugge** [pay-rails/pay#753]
I just started to implement Pay with Stripe, but `Pay::Webhooks::ProcessJob` (called with `"gid://namedrop/Pay::Webhook/4"`) is returning the following error:

```
NoMethodError: undefined method `charges' for #<Stripe::PaymentIntent:0x4dbc id=pi_3MEaBRIsfnFM3KWJ1Si42BTp>
JSON: { "id": "pi_3MEaBRIsfnFM3KWJ1Si42BTp", "object": "payment_intent", "amount": 390`
```

It turns out that this line no longer works:

https://github.com/pay-rails/pay/blob/eab715bd24ee3031b45102d10cf7c03ab5d3c590/lib/pay/stripe/webhooks/checkout_session_completed.rb#L12

As of November 16, 2022 the `Stripe::PaymentIntent` no longer has a `charges` method:

https://stripe.com/docs/changelog#november-16,-2022

What's the most recent API version the Stripe integration was tested with? Thanks in advance!

**Comment by marckohlbrugge** [comment]
I think the issue can be solved by replacing:

https://github.com/pay-rails/pay/blob/eab715bd24ee3031b45102d10cf7c03ab5d3c590/lib/pay/stripe/webhooks/checkout_session_completed.rb#L10-L15

With this:

```ruby
 if (payment_intent_id = event.data.object.payment_intent)   
  charges = ::Stripe::Charge.list({payment_intent: payment_intent_id})
  charges.each do |charge| 
     Pay::Stripe::Charge.sync(charge.id, stripe_account: event.try(:account)) 
   end 
 end
```

It does remove the `{stripe_account: event.try(:account)}.compact)` filter check. Not sure if you need it. If you do, then you're better off keeping the `payment_intent = …` and using `payment_intent.id` instead of `payment_intent_id`

**Comment by excid3** [comment]
Whoops, I missed that when updating the API versions recently. Thanks @marckohlbrugge

**Comment by excid3** [comment]
This should be fixed now as well @marckohlbrugge. I'll cut a new release shortly. 👍

**Comment by excid3** [comment]
Released 6.1.1
